src/Http/Cookie.php

```diff
@@ -63,11 +63,15 @@ public static function set(string $key, string $value, array $options = []): boo
     /**
      * Calculates the lifetime for a cookie
      *
-     * @param int $minutes Number of minutes or timestamp
+     * @param int|float $minutes Number of minutes or timestamp
      * @return int
      */
-    public static function lifetime(int $minutes): int
+    public static function lifetime($minutes): int
```

Comment:
Removed `int` type because it converts to float if the server doesn't support it.

Reply:
What converts to float?

Reply:
a number greater than the maximum integer, for 32-bit server on:

    Cookie::lifetime(253402214400); // you'll get type error

Reply:
To be honest I wouldn't allow floats, it would create a type mess that's very hard to test for edge-cases.

Have you tried the [solution I proposed here](https://github.com/getkirby/kirby/issues/2968#issuecomment-738266254) directly in the `forever()` method?

Reply:
It works, but I wanted to combine it because the `lifetime()` method has the same issue. Of course, `forever()` Kirby originated, `lifetime()` is a user issue.

Reply:
That's a good point! I'm kind of split here – your original solution definitely fixes the issue well, but I'm not sure if we should weaken our type hints just for that edge-case. If we could already use union types (`int|float`) in PHP 7.3, it would be a non-issue.